modules/gapi/include/opencv2/gapi/cpu/gcpukernel.hpp

```diff
@@ -248,6 +248,12 @@ struct OCVCallHelper<Impl, std::tuple<Ins...>, std::tuple<Outs...> >
             Impl::run(std::forward<Inputs>(ins)..., outs...);
             postprocess(outs...);
         }
+
+        template<typename... Outputs>
+        static void call(const Impl& impl, Inputs&&... ins, Outputs&&... outs)
+        {
+            impl(std::forward<Inputs>(ins)..., outs...);
+        }
```

Comment:
But I can't use template for recognize object type for `callFunctor` because I can't to pass template function to `std::bind`.

Of course I can make a wrapper.